src/styles/main.css

```diff
@@ -0,0 +1,84 @@
+body {
+  margin: 0;
+}
+
+.header {
+  margin: 20px auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  display: -webkit-box;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.logo {
```

Comment:
Styling for card, logo, header and other BEM blocks should be placed in separate files for each block
![image](https://user-images.githubusercontent.com/96409563/207928589-651e1277-117d-464c-98b2-a61051b144e9.png)